### ISSUE TITLE
Why not send text/files through chat conversations?

### ISSUE THREAD

**Issue by userzzzq** [localsend/localsend#462]
### Is your feature request related to a problem? Please describe.

LocalSend automatically converts text messages to txt files, that's not good. Why not send text/files through chat conversations? (like WeChat, Feem, etc) , that will obviously get a better user experience.

### Describe the solution you'd like

send text/files through chat conversations? (like WeChat, Feem, etc)

**Comment by jolestar** [comment]
I also like this feature. All the local devices can join a group, and send files or texts in that group like a broadcast.

**Comment by ElonSatoshi** [comment]
I would love to be able to set up a hotspot that anybody can join, install Localsend, and chat on.

**Comment by Tienisto** [comment]
WeChat and other services have a third party server that manages that all devices receive the files and messages even when they are offline.
This is not possible in LocalSend that is designed to be completely autonomous.

However, it is possible to create a local temporary chat room. Nevertheless, This app is called LocalSend and not LocalChat so I don't know if this would be out of scope of LocalSend.

**Comment by wayward777** [comment]
Agreed completely with the suggestion. The way it is currently set up is very clunky and takes a lot of clicks/taps to accomplish simple tasks.  Chat conversations view is IDEAL for this.  All the pieces are in place to do this already.

Some wishlist items for conversation mode:
* Paste images from clipboard into the input field
* Drag and drop file attachments into input field.
* Rich text
* Clickable links in chat view
* Configurable options for message persistance & message expiry

There could also be a "chatroom" function where you can join multiple devices to for multi-device communication.  Individual device-to-device communication in this scenario would effectively be like a "private message".

**Comment by yuehanwali** [comment]
Agree

**Comment by mohammadskabir** [comment]
Awesome. Some LAN apps do support chatting. Add a VPN to the mix and it can be a chat app too.

**Comment by mimodavila-ops** [comment]
> ### Is your feature request related to a problem? Please describe.
> Whether sending files or text, LocalSend is not user-friendly. Send text: LocalSend automatically creates a txt file, it's not necessary. When the file transfer is completed, the "sending file" page is always displayed, I have to click "done" to done .It complicates the simple thing. Why not send text or files via chat conversations? (like WeChat, Feem, etc) , that will obviously get a better user experience.
> 
> ### Describe the solution you'd like
> send text or files via chat conversations (like WeChat, Feem, etc)
> 
> ![111](https://github.com/localsend/localsend/assets/69742519/e90321d1-e88a-4c24-821d-c1aa96e964a0)

462

**Comment by NingInk** [comment]
<img width="1362" height="730" alt="Image" src="https://github.com/user-attachments/assets/1598b5ce-a594-4ba5-8d7c-803bbb00d92f" />
不用改原来的功能，选择附近的设备后，右边扩展出一个会话窗口（变成现在常用的三段式窗口）

**Comment by gfl43r9nd** [comment]
This already exists?
